docs/en/02_Developer_Guides/00_Model/06_SearchFilters.md

```diff
@@ -36,6 +36,10 @@ These suffixes can also take modifiers themselves. The modifiers currently suppo
 comparison uses the database's default. For MySQL and MSSQL, this is case-insensitive. For PostgreSQL, this is 
 case-sensitive.
 
+Note that all search filters (e.g. `:PartialMatch`) must be written in title case,
```

Comment:
OK, I've misunderstood the existing behaviour, but verified now that `MyField:case` is indeed already a feature (https://github.com/silverstripe/silverstripe-framework/blob/master/tests/model/DataListTest.php#L722). In that case, yeah ... I agree with your solution. The alternative would be either introducing new syntax (other than `:`), or using a more explicit object structure as Matt suggests. Neither of these are MVP here, so I'm good with the fix as-is.